pyirf/perf/irf_maker.py

```diff
@@ -409,19 +411,55 @@ def make_effective_area(
             )
 
             area[ibin] = (sel / simu_evts) * area_simu.value
-            energ_lo[ibin] = emin.value
-            energ_hi[ibin] = emax.value
+            energy_true_lo[ibin] = emin.value
+            energy_true_hi[ibin] = emax.value
 
-        t = Table()
-        t["ENERG_LO"] = Column(
-            energ_lo, unit="TeV", description="energy min", format="E"
+        table_energy = Table()
+        table_energy["ETRUE_LO"] = Column(
+            energy_true_lo,
+            unit="TeV",
+            description="energy min",
+            format=str(len(energy_true_lo)) + "E",
         )
-        t["ENERG_HI"] = Column(
-            energ_hi, unit="TeV", description="energy max", format="E"
+        table_energy["ETRUE_HI"] = Column(
+            energy_true_hi,
+            unit="TeV",
+            description="energy max",
+            format=str(len(energy_true_hi)) + "E",
         )
-        t[hdu_name] = Column(area, unit="m2", description="Effective area", format="E")
 
-        return IrfMaker._make_hdu(hdu_name, t, ["ENERG_LO", "ENERG_HI", hdu_name])
+        # Needed for format, a bit hacky...
```

Comment:
Why these two intervals?

Reply:
I f you are doing point-like IRFs, theta bins should have exactly one entry which is the distance of the on region to the camera center (0 for on, wobble distance for wobble observations).

Reply:
No specific reason for these values but a range is required for gammapy to work even though for now we are only working with point source MC.
We'll need to make a cleaner version later to work on diffuse.
Maybe add more comments here @JouvinLea ?